app/views/examples/translated/index.njk

```diff
@@ -0,0 +1,871 @@
+            href: "#",
+            text: "Genedigaethau, marwolaethau, priodasau a gofal"
+          },
+          {
+            href: "#",
+            text: "Busnes a hunan-gyflogedig"
+          }
+        ]
+      },
+      {
+        title: "Adrannau a pholisi",
+        width: "one-third",
+        items: [
+          {
+            href: "#",
+            text: "Sut mae llywodraeth yn gweithio"
+          },
+          {
+            href: "#",
+            text: "Adrannau"
+          },
+          {
+            href: "#",
+            text: "Ar draws y byd"
+          }
+        ]
+      }
+    ],
+    meta: {
+      items: [
+        {
+          href: "#",
+          text: "Cwcis"
+        },
+        {
+          href: "#",
+          text: "Cysylltwch"
+        },
+        {
+          href: "#",
+          text: "Telerau ac Amodau"
+        }
+      ],
+      html: 'Adeiladwyd gan y <a href="#" class="govuk-footer__link">Government Digital Service</a>'
+    },
+    contentLicence: {
+      html: "Mae’r holl gynnwys ar gael o dan y <a href='#' class='govuk-footer__link'>Drwydded Llywodraeth Agored v3.0</a>, ac eithrio lle nodir yn wahanol"
+    },
+    copyright: {
+      html: "<span>Hawlfraint y Goron</span>"
+    }
+  }) }}
+
+  {{ govukHeader({
+    homepageUrl: "#",
+    serviceName: "Service name",
+    serviceUrl: "#",
+    menuButtonLabel: "Dangos neu guddio'r ddewislen",
+    menuButtonText: "Dewislen",
+    navigation: [
+      {
+        href: "#1",
+        text: "Eitem llywio 1",
+        active: true
+      },
+      {
+        href: "#2",
+        text: "Eitem llywio 2"
+      },
+      {
+        href: "#3",
+        text: "Eitem llywio 3"
+      },
+      {
+        href: "#4",
+        text: "Eitem llywio 4"
+      }
+    ]
+  }) }}
+
+  {{ govukInsetText({
+    text: "Gall gymryd hyd at 8 wythnos i gofrestru atwrneiaeth arhosol os nad oes unrhyw gamgymeriadau yn y cais."
+  }) }}
+
+  {% set notificationBannerHtml %}
+    <p class="govuk-notification-banner__heading">
+      Mae gennych 7 diwrnod ar ôl i anfon eich cais.
+      <a class="govuk-notification-banner__link" href="#">Gweld cais</a>.
+    </p>
+  {% endset %}
+
+  {{ govukNotificationBanner({
+    html: notificationBannerHtml,
+    titleText: "Pwysig"
+  }) }}
+
+  {{ govukPagination({
+    previous: {
+      text: "Blaenorol",
+      href: "#"
+    },
+    next: {
+      text: "Nesaf",
+      href: "#"
+    },
+    items: [
+      {
+        number: 1,
+        href: "#",
+        visuallyHiddenText: "Tudalen 1"
+      },
+      {
+        number: 2,
+        current: true,
+        href: "#",
+        visuallyHiddenText: "Tudalen 2"
+      },
+      {
+        number: 3,
+        href: "#",
+        visuallyHiddenText: "Tudalen 3"
+      }
+    ]
+  }) }}
+
+  {{ govukPanel({
+    titleText: "Cais wedi'i gwblhau",
+    html: "Eich rhif cyfeirnod<br><strong>HDJ2123F</strong>"
+  }) }}
+
+  {{ govukPhaseBanner({
+    tag: {
+      text: "alpha"
```

Comment:
Do we want to translate this?

Reply:
Unsure about this one, as they're like official "statuses" for a service. Examples I've found don't tend to translate it, e.g:

https://www.manage-student-loan-balance.service.gov.uk/ors?locale=cy
https://www.passport.service.gov.uk/filter/overseas?lang=cy&_ga=2.191870766.509248048.1661854651-1299619278.1657288777

Reply:
Yep, fair enough. I think we already have an example of a translated tag elsewhere on the page anyway.